### PR TITLE
Updated files to typescript

### PR DESCRIPTION
**Pull request by 1x2kb**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -31,7 +28,7 @@ app.use(function(req, res, next) {
 });
 
 // error handler
-app.use(function(err, req, res, next) {
+app.use(function(err: any, req: any, res: any, next: any) {
```
Comment (bradrf) [inline app.ts:31]: might be good to be a little more precise and do something like:
```javascript
import express, { Request, Response, NextFunction } from 'express';
…
app.use(function(err: Error, req: Request, res: Response, next: NextFunction) {
```

Comment (bradrf) [reply app.ts:31]: this aids in code completion in various editors as well as catching silly misuses of those variables within the function

File: app.ts [hunk 2]
```diff
@@ -1,15 +1,12 @@
-var createError = require('http-errors');
-var express = require('express');
-var path = require('path');
-// var cookieParser = require('cookie-parser');
-var logger = require('morgan');
+import createError from "http-errors";
```
Comment (bradrf) [inline app.ts:1]: this is total nitpick, but it'd be good to be consistent with quotes. in general, it's a "best practice" to use single quotes in most circumstances

File: routes/eth-lookup.ts [hunk 3]
```diff
@@ -1,23 +1,27 @@
-var express = require('express');
-var router = express.Router();
-const { providers } = require('ethers');
-const redis = require('../modules/redis');
+// var express = require('express');
+import express from "express";
+const router = express.Router();
+// const { providers } = require('ethers');
+  import { providers } from "ethers";
+// const redis = require('../modules/redis');
```
Comment (bradrf) [inline routes/eth-lookup.ts:6]: we can remove these requires, now, yes?

**Review by bradrf — APPROVED** [review]

File: routes/eth-lookup.ts (hunk 3 above)
Comment (1x2kb) [reply routes/eth-lookup.ts]: Removed

File: app.ts (hunk 2 above)
Comment (1x2kb) [reply app.ts:1]: I'll add a linter shortly and it'll take care of that problem.

Comment (1x2kb) [reply app.ts:1]: Going to ignore for now

**Pull request merged**